Makefile

```diff
@@ -84,30 +89,16 @@ image_bins: init
 	if [ ${SUPPORT_PLUGINS} = "yes" ];then\
 		CC=${CC} CGO_ENABLED=1 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-scheduler ./cmd/scheduler;\
 	else\
-	 	CC=${CC} CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-scheduler ./cmd/scheduler;\
-  	fi;
+		CC=${CC} CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-scheduler ./cmd/scheduler;\
+	fi;
 
-images: image_bins
+images:
```

Comment:
the `image_bins` output is same as `make vc-scheduler/vc-controller-manager/vc-webhook-manager` output?

Looks like it not same from my real test.

Reply:
The output should be the same on the same platform. What's your test about?

Reply:
https://github.com/volcano-sh/volcano/pull/2049#discussion_r841092824

I was using the image_bins with buildx it works, but I was using current  implementations (make vc-xxx, go build) with buildx, it doesn't work.

so I think might be binary problem.